tests/filesystem/test_filesystem.py

```diff
@@ -16,3 +20,13 @@ def test_content_mimetype(png_image, jpg_image):
 
     with open(jpg_image, "rb") as fh:
         assert get_content_mimetype(fh.read(64)) == "image/jpeg"
+
+
+@pytest.mark.parametrize(
+    "file_name, expected_mime", [("star.svg", "image/svg+xml")],
+)
+def test_mime_overrides(file_name, expected_mime):
+    fpath = pathlib.Path(__file__).parent.parent.joinpath("files").joinpath(file_name)
```

Comment:
Can we reuse a common fixture so we don't have to to this kind of fragile stuff in the test itself?